pages/PlaceOrInstitution.xml

```diff
@@ -42,6 +44,15 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
             <p>modify those references that actually point to a repository
                 and not to a place, if any.</p>
         </item>
+        <item>
+            <p>you can link the place and repository by adding a relation of the skos:exactMatch type
```

Comment:
I think it would be better to explicitly give the ID of the example record

Reply:
ok done